.storybook/main.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     addons: ['@storybook/preset-typescript'],
     webpackFinal: async (config, { configType }) => {
         config.module.rules.push({
-            test: /\.tsx?$/,
```

Comment:
Because we're importing CMP now, Storybook was trying to build using js files that it wasn't configured to transform